### ISSUE TITLE
Support full set of latent awakenings

### ISSUE THREAD

**Issue by weschao** [madcowfred/padherder-issues#1042]
This would be particularly useful for people who use padherder in connection with padspike, so that killer latents are automatically applied.

**Comment by weschao** [comment]
(I'd be happy to work on this, if you wanted another contributor.)

**Comment by ryuujinzero** [comment]
First step would be figuring out how to properly implement the double wide latents (the killers and ++ latents). This is kinda where we've been stuck for a while, but still working on it.

**Comment by weschao** [comment]
Do the latents actually need to be double-wide, or is there a different problem? I had a similar problem with PadSpike and decided to just treat double-wide latents as single-wide.

A user who really wants to could incorrectly add more than 3 double-wide latents, but I feel like having the functionality would be worth that minor inelegance.

**Comment by ryuujinzero** [comment]
That was one point where we were divided. Some of us didn't care whether you could erroneously add incompatible latent killers, while others wanted the latent greyed out if not compatible. If treating the latents as single wide, how would the damage boost be calculated in teams? What if you only select one single wide god killer, 3 single wide dragon killers, and 1 single wide balance killer? Something would probably break.

**Comment by weschao** [comment]
I think preventing a latent from being added if incompatible is nice, but not necessary (and would strongly prefer to allow incompatible latents if it means getting the functionality earlier).

Do killer latents actually have any effect within PadHerder itself? PadSpike already properly handles an arbitrary number of killer latents -- if you added 6 single-wide dragon killer latents, for example, it would give you the 11.39x boost for dragons.

**Comment by Exaxxion** [comment]
I would agree with weschao. Having 6th and killer latent slots available even if you can do invalid things with it is highly preferable to not having the feature at all.
